src/coreComponents/physicsSolvers/NonlinearSolverParameters.cpp

```diff
@@ -187,44 +188,50 @@ void NonlinearSolverParameters::postProcessInput()
                         getWrapperDataContext( viewKeysStruct::lineSearchResidualFactorString() ) << ": should be positive" );
 
   if( getLogLevel() > 0 )
-  {
```

Comment:
I was once told that this is part of the geos coding style. i.e. brackets even for a single statement.